mxcube3/routes/SampleCentring.py

```diff
@@ -601,8 +601,29 @@ def accept_centring():
 
 @mxcube.route("/mxcube/api/v0.1/sampleview/centring/reject", methods=['PUT'])
 def reject_centring():
-    """
-    Reject the centring position.
-    """
+    """Reject the centring position."""
     mxcube.diffractometer.rejectCentring()
     return Response(status=200)
+
+
+@mxcube.route("/mxcube/api/v0.1/sampleview/moveto", methods=['PUT'])
+def pos_from_x_y():
```

Comment:
Why would we need to move to center of sample video ? Normally we want to move to beam (which is the center in your case). Maybe there is something I don't see please tell me if we need to move to center ?